client/ladder/action-creators.ts

```diff
@@ -97,6 +208,6 @@ export function getInstantaneousSelfRank(spec: RequestHandlingSpec<void>): Thunk
 }
 
 /** Navigates to the ladder standings (optionally for a given matchmaking type). */
-export function navigateToLadder(type?: MatchmakingType, transitionFn = push) {
-  transitionFn(urlPath`/ladder/${type ?? ''}`)
+export function navigateToLadder(type?: MatchmakingType, seasonId?: SeasonId, transitionFn = push) {
+  transitionFn(urlPath`/ladder/${type ?? ''}` + (seasonId ? urlPath`/${seasonId}` : ''))
```

Comment:
This doesn't really seem correct, it is possible to not specify a matchmaking type but specify a season ID with this API and that generates URLs that are nonsensical